.eslintrc.cjs

```diff
@@ -139,6 +139,8 @@ module.exports = {
                     },
                 ],
                 "no-extra-boolean-cast": "error",
+                // to avoid unnecessary runtime dependencies between files
+                "@typescript-eslint/consistent-type-imports": ["error", { fixStyle: "inline-type-imports" }],
```

Comment:
Should we add this to https://github.com/matrix-org/eslint-plugin-matrix-org/blob/a1ce5fbb7d425b932c504ae2bb7f985e3c07d9fd/typescript.js#L99 instead?

Reply:
That would make sense to me, assuming agreeable that this is generally a Good Thing.

If so I can make the necessary PRs.

Reply:
FTR, the upstream rule was added in https://github.com/matrix-org/eslint-plugin-matrix-org/pull/44